oca_dependencies.txt

```diff
@@ -0,0 +1,13 @@
+List of OCA project dependencies for shopwareerpconnect
+========================================================
+
+1)	account_payment_mode
+2)	account_payment_partner
+3)	account_payment_sale
+4)	connector
+5)	connector_base_product
+6)	connector_ecommerce
+7)	product_multi_category
+8)	sale_automatic_workflow
+9)	sale_automatic_workflow_payment_mode
+10)	sale_exception
```

Comment:
This is not the format of oca_dependencies.txt! 
See https://github.com/OCA/sale-workflow/blob/11.0/oca_dependencies.txt

Reply:
mention repo, not the module name!

Reply:
Yes of course, I will add repositories instead of modules